08-mapping.Rmd

```diff
@@ -905,12 +905,89 @@ One potential problem with the ease of developing prototypes with **shiny** is t
 For that reason, despite advocating **shiny**, we recommend starting with the longer established technology of a pen and paper as the first stage for interactive mapping projects.
 This way your prototype web applications should be limited not by technical considerations, but by your motivations and imagination.
 
-```{r coffeeApp-html, echo=FALSE, message=FALSE, fig.cap="coffeeApp, a simple web mapping application for exploring global coffee production in 2016 and 2017. Interactive version available online at geocompr.robinlovelace.net.", fig.scap="coffeeApp, a simple web mapping application.", eval=knitr::is_html_output()}
-knitr::include_app("https://shiny.robinlovelace.net/coffeeApp/")
+```{r CycleHireApp-html, echo=FALSE, fig.cap="Hire a cycle App, message=FALSE," a simple web mapping application for finding the closest cycle hiring station based on your location and requirement of cycles. Interactive version available online at geocompr.robinlovelace.net.", fig.scap="Cycle Hire App, a simple web mapping application.", eval=knitr::is_html_output()}
+
```

Comment:
Heads-up @KiranmayiV the code here is causing the CI checks to fail. Can you remove these lines of code?

Reply:
yup, done.

Reply:
sorry my bad, there was a error in the chunk's caption. I corrected it. However the check are still failing.